nodejs/README.md

```diff
@@ -0,0 +1,208 @@
+application implements a generic contract with the platform. This contract
+allows it to be scaled and managed by the platform without intimate knowledge of
+how the application works.
+
+[cloud.gov](https://cloud.gov/) is the preferred platform at 18F. Building your
+Node.js application with the Twelve-factor methodology in mind will ensure your
+application is designed for cloud.gov.
+
+
+### Best practices
+
+#### Recommend
+
+- Build your application on top of cloud.gov.
+- Read the [Twelve-factor methodology](https://12factor.net/) and keep it in
+  mind as you build your application.
+
+
+## Choosing a version
+
+### Node.js
+
+Node.js supports certain versions as Long-term Support (LTS) versions. We
+recommend the [current active LTS](https://github.com/nodejs/LTS#lts-schedule1)
+version, currently v6.x.
+
+
+### ECMAScript
+
+ES5, ES6, ES7? JavaScript is an evolving language and is defined by
+[ECMAScript](https://en.wikipedia.org/wiki/ECMAScript). Modern browsers support
+ES5, but when writing code on the server, you don't have to write
+browser-compatible code. With `--harmony`, Node.js [supports many ES6 and ES7
+features](http://node.green/).
+
+You can also use the same build tools that you use on the front end, like
+webpack and browserify with babel, to transpile your code. This allows you to
+fill in the feature gaps that are not yet supported by Node.js. This also means
+your browser code and server code use the same ECMAScript features.
+
+
+### Best practices
+
+
+#### Recommend
+
+- Use the [current active LTS](https://github.com/nodejs/LTS#lts-schedule1)
+  version of Node.js.
+
+
+#### Consider
+
+- What syntax are you using on the front end? Consider using the same syntax and
+  features for both the  front end and back end code.
+- Many developers write JavaScript in the browser (ES5) even if they aren't
+  a JavaScript developer. Using a syntax that many folks are familiar with
+  allows more folks to jump in.
+- Front end build tools make it easy to choose a syntax you want and polyfill
+  the missing features in the browser.
+
+
+## Frameworks
+
+Frameworks provide a common baseline for your application. ATO requires
+configurations that most frameworks provide out of the box, or make it very easy
+to enable.
+
+The most common framework at 18F is [Express][expressjs]. You don't have to
+choose Express, but you should use an existing framework with an active
+community for support.
+
+Here are some existing projects at 18F using Express:
+
+- [Federalist][github-federalist]
+- [OMB eRegs][github-omb-eregs]
+- [Analytics Reporter API][github-analytics-report]
+
+You can [find many
+more](https://github.com/search?q=org%3A18F+filename%3Apackage.json+express&type=Code&utf8=%E2%9C%93) internal and
+external projects as well.
+
+
+### Best practices
+
+
+#### Recommend
+
+- Use an existing framework with active community support.
+
+
+#### Suggest
+
+- Use [Express][expressjs].
```

Comment:
I bet you're trying to avoid having too many specific libraries/packages listed in this guide, but maybe consider adding Hapi to either this list of suggestions, or to a list of considerations for this section.